bloom-housing/bloom
# feat: uptake field value v2

# Pull Request Template

## Issue Overview

This PR addresses #3396


## Description

This PR uptakes the new FieldValue component from seeds.
[ZeroHeight Docs](https://zeroheight.com/5e69dd4e1/p/715019-field-value/b/93a6d7)
[Storybook](https://storybook-ui-seeds.netlify.app/i)

## How Can This Be Tested/Reviewed?

As compared to dev, the styling is subtly different intentionally in terms of the font size, color, and padding.

To test this PR, check out the new component in the following places:
* Partners listing detail page
* Partners application detail page
* Public application add household members page
* Public application summary page

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
